provider/https/file/file.go

```diff
@@ -0,0 +1,116 @@
+package httpsFile
```

Comment:
you must use the folder name

Reply:
I don't understand `provider/https/file/file.go`.

Why not `provider/https/https.go` like other provider?

Reply:
Because, the subpackage `provider/https` will contain other HTTPS providers (entrypoints, acme...) in the future.